sormas-api/src/main/java/de/symeda/sormas/api/utils/LocationHelper.java

```diff
@@ -95,4 +95,11 @@ public static boolean checkIsEmptyLocation(LocationDto location) {
 
 		return true;
 	}
+
+	public static void resetContinentFieldsIfCountryRemoved(LocationDto newLocation, LocationDto oldLocation) {
```

Comment:
I believe the Locations that are already saved in database in a state that now is prevented should be cleaned up as well.

Reply:
Since a state in which the continent is present, but the country is not, can be valid for locations (exposures and activities as case), I don't think we can do that without potentially messing up existing data.